common/app/model/EmailNewsletters.scala

```diff
@@ -227,10 +227,10 @@ object EmailNewsletters {
     teaser = "Everything you need to know from the Guardian's music site, squeezed into one handy email",
     description = "Everything you need to know from the Guardian's music site, squeezed into one handy email.",
     frequency = "Every Friday",
-    listId = 39,
-    aliases = List(3834, 3835),
+    listId = 3834,
```

Comment:
It might make sense to switch the primary listId based on the AB test switch, so that the new list ID is only enabled when the test is enabled.